lte/gateway/deploy/roles/magma/files/nx_actions.py

```diff
@@ -215,8 +215,8 @@ def parse(cls, buf):
         def serialize(self, buf, offset):
             data = self.serialize_body()
             payload_offset = (
-                    ofp.OFP_ACTION_EXPERIMENTER_HEADER_SIZE +
-                    struct.calcsize(NXAction._fmt_str)
+                ofp.OFP_ACTION_EXPERIMENTER_HEADER_SIZE +
```

Comment:
should we disable `W504`?

Reply:
good call! will do this as a followup

Reply:
https://github.com/magma/magma/pull/11880